content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+
+#### Xtensa
+
+- [ESP32](https://nuttx.apache.org/docs/latest/platforms/xtensa/esp32/index.html#peripheral-support)
+- [ESP32-S2](https://nuttx.apache.org/docs/latest/platforms/xtensa/esp32s2/index.html#peripheral-support)
+- [ESP32-S3](https://nuttx.apache.org/docs/latest/platforms/xtensa/esp32s3/index.html#peripheral-support)
+
+#### RISC-V
+
+- [ESP32-C3](https://nuttx.apache.org/docs/latest/platforms/risc-v/esp32c3/index.html#peripheral-support)
+- [ESP32-C6](https://nuttx.apache.org/docs/latest/platforms/risc-v/esp32c6/index.html#peripheral-support)
+- [ESP32-H2](https://nuttx.apache.org/docs/latest/platforms/risc-v/esp32h2/index.html#peripheral-support)
+
+### Installing
+
+{{< alert >}}
+The following steps use Ubuntu 22.04 as an example. Please check the [official documentation](https://nuttx.apache.org/docs/latest/quickstart/install.html#prerequisites) for other Operating Systems.
+{{< /alert >}}
+
+#### Installing System Dependencies
+
+Install the system dependencies required to build NuttX:
+
+``` bash
+sudo apt install \
+automake binutils-dev bison build-essential flex g++-multilib gcc-multilib \
+genromfs gettext git gperf kconfig-frontends libelf-dev libexpat-dev \
+libgmp-dev libisl-dev libmpc-dev libmpfr-dev libncurses5-dev libncursesw5-dev xxd \
+libtool picocom pkg-config python3-pip texinfo u-boot-tools util-linux
+```
+
+Also, please install some python packages:
+
+```bash
+pip install kconfiglib
+```
+
+#### Espressif Tools and Toolchains
+
+Once the NuttX dependencies are available, it’s time to install the compilation tools for Espressif devices.
+
+Install `esptool`:
+
+```bash
+pip install esptool==4.8.dev4
+```
+Finally, install the toolchains to build the Espressif SoCs. It can be installed depending on the SoC being used, but it is highly recommended to keep all of them installed.
+
+##### **Xtensa Devices (ESP32, ESP32-S2, ESP32-S3)**
+
+Each Xtensa-based device has its own toolchain, which needs to be downloaded and configured separately. All the toolchains used by NuttX can be seen in the Docker container used for upstream testing. It is highly recommended to check the updated versions according to the file in https://github.com/apache/nuttx/tree/master/tools/ci/docker/linux/Dockerfile
+
+###### **ESP32**:
+
+Create a directory to install the toolchain. It can be on your home directory:
+```bash
+mkdir -p ~/xtensa-esp32-elf-gcc
+```
+
+Download and extract the toolchain on the new directory:
+```bash
+cd ~/xtensa-esp32-elf-gcc
+wget https://github.com/espressif/crosstool-NG/releases/download/esp-12.2.0_20230208/xtensa-esp32-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+tar -xf xtensa-esp32-elf-12.2.0_20230208-x86_64-linux-gnu.tar.xz
+```
+
+Once extracted, add the location of the binaries to the user’s PATH and reload the terminal:
+
+```bash
+echo "export PATH=$HOME/xtensa-esp32-elf-gcc/xtensa-esp32-elf/bin:\$PATH" >> ~/.bashrc
+```
+
+###### **ESP32-S2**:
```

Comment:
Is ESP32-S2 important in the NuttX context? Basically it has been phased out by ESP32-S3 and any new development begins on S3 or newer chips. Does it bring value to have S2 here?

Reply:
We still provide support for S2, so I think it's worth keeping it in the article...